changes/ce/fix-10190.en.md

```diff
@@ -0,0 +1 @@
+Fix the issue where nodes responses to the list bridges RPC were incorrectly flattened, which caused List Bridges API HTTP handler to crash when there was more than 1 node in the cluster.
```

Comment:
no need for the changelog since the bug is not released yet.

Reply:
same for the master branch PR.

Reply:
#10203